sodetlib/operations/tracking.py

```diff
@@ -506,7 +510,10 @@ def relock_tracking_setup(S, cfg, bands, reset_rate_khz=None, nphi0=None,
 
     # Choose frac_pp to be the mean of all running bands.
     # This is the frac-pp at the flux-ramp-rate used for optimization
-    fpp0 = np.median(frac_pp0)
+    fpp0 = np.mean(frac_pp0)
```

Comment:
I switched this to mean, since we were frequently with modules where the north and south side had different optimal frac-pp. If bands 0-3 want a frac-pp of 0.3, and bands 4, 5, 6 want 0.4, we want to choose something between 0.3 and 0.4 instead of choosing 0.3. I think if there is a large outlier it should just not be included, or you should choose the frac-pp value manually

Reply:
Ok. Can we flag for large outliers then since that should indicate a failure that requires further investigation on that band?

Reply:
Why don't we not include that in this PR and add it as a new PR that you can test in P10R2.